dart/utils/sdf/SdfParser.cpp

```diff
@@ -1258,7 +1258,7 @@ static void readAxisElement(
     const Eigen::Isometry3d& _parentModelFrame,
     Eigen::Vector3d& axis,
     double& lower, double& upper, double& initial, double& rest,
-    double& damping)
+    double& damping, double &friction, double &sprint_stiffness)
```

Comment:
```suggestion
    double& damping, double& friction, double& sprint_stiffness)
```

Reply:
Applied `clang-format` to `SdfParser.cpp` overall.